planet.rb

```diff
@@ -0,0 +1,18 @@
+class Planet
+  attr_reader :name, :color, :mass_kg, :distance_from_sun_km, :fun_fact
+
+  def initialize (name, color, mass_kg, distance_from_sun_km, fun_fact)
+    @name = name
+    @color = color
+    raise ArgumentError.new("Not a valid number for the mass") if mass_kg.to_i <= 0
```

Comment:
Extremely minor thing:  It would be more readable to have these if - raise lines at the top of `initialize` and put the assignments below.